### PR TITLE
Create 2016-01-12-instructor-debriefing-round-01.html

### PR DESCRIPTION
**Pull request by raynamharris**
Here is a brief write up of this morning's debriefing. I'd like to give @k8hertweck a chance to add/make changes as necessary (is the best practices slide show the link you found as well or is there something else?). 

Also, I used the template made by @rgaiacs, but modified it by deleted the section on installation issues (not discusses), changed the "what went wrong" to "what could have gone better", and added an "other comments" section because I wasn't sure where the link to the workshop blog should go.

### CODE REVIEW

File: _posts/2016/01/2016-01-12-instructor-debriefing-round-01.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+root: ../../..
+author: "Rayna Harris and Kate Hertweck"
+title: "2016 Post-Workshop Instructor Debriefing, Round 01"
+date: 2016-01-12
+time: "10:00:00"
+category: ["Debriefing"]
+---
+<!-- start excerpt -->
+<p>
+On January 11, 2016 we ran the 1st round of post-workshop instructor debriefing of the calendar year! The mentoring committee ran 22 debriefing session in 2015, and we are exciting about continuing to host and improve these session this year.
```
Comment (k8hertweck) [inline _posts/2016/01/2016-01-12-instructor-debriefing-round-01.html]: sessions (plural)

File: _posts/2016/01/2016-01-12-instructor-debriefing-round-01.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+<p>
+On January 11, 2016 we ran the 1st round of post-workshop instructor debriefing of the calendar year! The mentoring committee ran 22 debriefing session in 2015, and we are exciting about continuing to host and improve these session this year. 
+
+Interestingly, all instructors present participated in 3-day format Software Carpentry workshops. Read on to find out more about the changes made, what worked well, and what didn't. 
+</p>
+<!-- end excerpt -->
+<h2>Worked well</h2>
+Both of these Nordic workshops were spearheaded by newly minted instructors. Greg Wilson had approached them in the past about becoming instructors and hosting workshops, and this effort has come to fruition. Both workshops were well attended and received, and everyone is looking forward to continuing learning, coding, and networking. 
+
+The stockholm group created an exercise/challenge that combined testing and pull requests. Students were given a broken python function; after fixing the errors, the student's submitted their changes via pull requests. View the exercise [here](https://github.com/bast/python-tdd-exercises).
```
Comment (k8hertweck) [inline _posts/2016/01/2016-01-12-instructor-debriefing-round-01.html]: capitalize Stockholm

File: _posts/2016/01/2016-01-12-instructor-debriefing-round-01.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+The Heliski group incorporated afternoon work sessions where the students could apply the newly-learned tools to their own data or get more help of the morning's topics.
```
Comment (k8hertweck) [inline _posts/2016/01/2016-01-12-instructor-debriefing-round-01.html]: Helsinki (here and below)

File: _posts/2016/01/2016-01-12-instructor-debriefing-round-01.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+The stockholm group created an exercise/challenge that combined testing and pull requests. Students were given a broken python function; after fixing the errors, the student's submitted their changes via pull requests. View the exercise [here](https://github.com/bast/python-tdd-exercises).
+
+The Heliski group incorporated afternoon work sessions where the students could apply the newly-learned tools to their own data or get more help of the morning's topics. 
+
+<h2>What could have gone better</h2>
+The Heliski group gave a lecture using the Software Carpentry [Best Practices in Scientific Computing slideshow](http://swcarpentry.github.io/slideshows/best-practices/index.html). The material is excellent, but the lecture style delivery was a stark constrast to the hands-on style of teaching the day before. 
+
+We wonder if there isn't a way to add the "rules" to various lesson where appropriate. This way, instructors can weave these best practices into the workshop, without needed a separate block of time to cover the. Thoughts?
```
Comment (k8hertweck) [inline _posts/2016/01/2016-01-12-instructor-debriefing-round-01.html]: "without needing a separate block of time to cover the slides independently."

Comment (wking) [inline _posts/2016/01/2016-01-12-instructor-debriefing-round-01.html]: “the.” → “them.”.

**Comment by k8hertweck** [comment]
Thanks so much for the draft! Only a few minor comments for spelling, etc. Are we adding extra content from the evening session?

**Comment by raynamharris** [comment]
No one attended the evening session, so there's nothing to add. 

I fixed the typos and such, so this is good to go.

**Comment by rgaiacs** [comment]
@raynamharris and @k8hertweck Thanks very much. :sparkling_heart: 

I made [minor changes](https://raw.githubusercontent.com/swcarpentry/website/gh-pages/_posts/2016/01/2016-01-12-instructor-debriefing-round-01.md) to the file to fix the compilation. And we now can use Markdown. :heart:

**Pull request merged**